make it possible to fully configure route middleware (#36)
cybex-gmbh/laravel-transmorpher-client: CHANGELOG.md

# Release Notes

## [v0.4.0](https://github.com/cybex-gmbh/laravel-transmorpher-client/compare/v0.3.0...v0.4.0)

> [!WARNING]
> Breaking change!

> - Route middleware is now fully configurable

Reviewer: ```suggestion
- Route middlewares are now fully configurable
```

  - the `SubstituteBindings` middleware is now always applied as it is necessary to resolve route model bindings

>   - the `web` and `auth` middlewares are now only applied by default when no config value is set

Reviewer: ```suggestion
  - when `transmorpher.routeMiddleware` is not set, the `web` and `auth` middlewares are applied
```
